tests/test_cli.py

```diff
@@ -1,17 +1,40 @@
-from my_app.cli import main
+from freezegun import freeze_time
 
+from cc_slips.cli import main
 
-def test_cli_no_options(caplog, runner):
-    result = runner.invoke(main)
+
+@freeze_time("2023-01-02")
+def test_cli_only_required_options(caplog, runner):
+    result = runner.invoke(
+        main, ["-s", "source@example.com", "-r", "recipient@example.com"]
+    )
     assert result.exit_code == 0
     assert "Logger 'root' configured with level=INFO" in caplog.text
-    assert "Running process" in caplog.text
+    assert "Starting credit card slips process" in caplog.text
     assert "Total time to complete process" in caplog.text
```

Comment:
Do we want to check for `Command called with...` log entry here as well for consistency?

Reply:
Nope, that gets logged at debug level so it won't show up in the logs here.